### ISSUE TITLE
4.2 issue? Unmatched patterns: OpSpread

### ISSUE THREAD

**Issue by cambiata** [haxe-utest/utest#109]
Getting the following error on 
- Haxe Compiler 4.2.0
- Haxe Compiler 4.2.0-rc.1+18fdc8c
(but not on Haxe Compiler 4.2.0-rc.1+2776f3cd4)

```
\.haxelib\utest/1,13,1/src/utest/utils/TestBuilder.hx:408: characters 17-19 : Unmatched patterns: OpSpread
\.haxelib\utest/1,13,1/src/utest/Test.hx:6: character 1 : Build failure
```

**Comment by cambiata** [comment]
Seems to be resolved in version 1.13.2.
Thanx!

**Comment by RealyUniqueName** [comment]
Ah, yes. 
Sorry, I forgot to reply to this issue )
